### PR TITLE
added final Not Found tests, fixed sound bad inputs issues in the ser…

### PR DESCRIPTION
**Pull request by Nhinds88**
…vices and added some system out prints

### CODE REVIEW

File: src/test/java/cst438flights/controller/CustomerRestControllerTest.java [hunk 1]
```diff
@@ -79,6 +79,27 @@ public void getCustomerPreviousFlightsTest() throws Exception {
         assertThat(resultFightInfo.toString()).isEqualTo(expectedFlightInfo.toString());
     }
 
+    @Test
```
Comment (espresso64) [inline src/test/java/cst438flights/controller/CustomerRestControllerTest.java:82]: Nice work adding the series of not founds tests

File: src/main/java/cst438flights/service/CustomerService.java [hunk 2]
```diff
@@ -24,6 +24,10 @@
 
         Customer customer = customerRepository.findByEmail(email);
 
+        if (customer == null) {
```
Comment (Anelon) [inline src/main/java/cst438flights/service/CustomerService.java:27]: Good call on returning null if the cutomer doesn't exist

File: src/test/java/cst438flights/controller/FlightRestControllerTest.java [hunk 3]
```diff
@@ -85,6 +81,26 @@ public void getFlightInfoArrival() throws Exception {
         assertThat(returnFlight).isEqualTo(expectedFlight);
     }
 
+    @Test
+    public void getFlightInfoArrivalNotFound() throws Exception {
```
Comment (espresso64) [inline src/test/java/cst438flights/controller/FlightRestControllerTest.java:85]: Nice work adding the not founds tests

File: src/test/java/cst438flights/controller/CustomerRestControllerTest.java [hunk 4]
```diff
@@ -115,11 +115,11 @@ public void getCustomerPreviousReservationsRestTest() throws Exception {
 
         given(customerService.getPreviousReservationsRest("jedi@lightside.com")).willReturn(reservationFlightInfoList);
```
Comment (espresso64) [inline src/test/java/cst438flights/controller/CustomerRestControllerTest.java:117]: Nice work, fixed testing issue

File: src/test/java/cst438flights/controller/CustomerRestControllerTest.java [hunk 5]
```diff
@@ -115,11 +115,11 @@ public void getCustomerPreviousReservationsRestTest() throws Exception {
...
 
-        MockHttpServletResponse response = mvc.perform(get("/api/previous_reservation/email")).andReturn().getResponse();
+        MockHttpServletResponse response = mvc.perform(get("/api/previous_reservation/jedi@lightside.com")).andReturn().getResponse();
```
Comment (espresso64) [inline src/test/java/cst438flights/controller/CustomerRestControllerTest.java:118]: nice work catching this bug, fixed testing issue

**Review by Anelon — APPROVED** [review]
Tests look good, better coverage for the code now

**Review by espresso64 — APPROVED** [review]
Code looks good, added tests working great. Nice work overall! Approved.

**Pull request merged**